.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig is awesome: https://EditorConfig.org
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,ts, *.json, *.yml}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
```

Comment:
I think the editorconfig plugin is not installed, it should add line breaks when saving the file
(I use VSCode, I had to add it)